src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb

```diff
@@ -705,5 +707,28 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
         End Class
 #End Region
 
+        Public Class SerializationProvider
+
+            Public Shared Sub Serialize(stream As Stream, value As Object)
+                If value Is Nothing Then
+                    Return
+                End If
+                Using writer As New BinaryWriter(stream, Encoding.UTF8, True)
```

Comment:
Using a named argument makes this clearer.

```suggestion
                Using writer As New BinaryWriter(stream, Encoding.UTF8, leaveOpen:=True)
```

Ditto below.

Reply:
Is this personal preference or a standard we do as a team? Over the last few years, I got used to [this feature](https://www.jetbrains.com/help/resharper/Inline_Parameter_Name_Hints.html#parameter-name-hints) of Resharper. Plus, usual code styling tools will make the `leaveOpen:=` grey out and say, "Remove unnecessary named parameter". It doesn't seem we have that styling, though. Basically, I've used named parameters when they are needed (to compile), not as code comments.

Reply:
I've added these either way. But, my initial intuition would always lead me to believe they are necessary, without looking at all the method overloads; if I was seeing this in a code review or in the code itself.